status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.125.0",
-    "commit-sha1": "e40cbfc28f6195b31eb0f59cbd58fd1d77a12821",
-    "src-sha256": "0nmys3mf46pilmqrr937b0bc1qi9h51aamyswngy4qfb96nvky85"
+    "version": "a2f7307cae06afa8770ac5b191770ed501338c14",
```

Comment:
revert?

Reply:
That's for QA testing, until the corresponding `status-go` PR is merged.